frontend/packages/shared/src/navigation/main-header/ProfileMenu/ProfileMenu.tsx

```diff
@@ -87,9 +87,7 @@ export const ProfileMenu = ({
       aria-haspopup
       ref={menuRef}
     >
-      <Paragraph as='span' size='small' className={classes.userOrgNames}>
-        {userNameAndOrg}
-      </Paragraph>
+      <div className={classes.userOrgNames}>{userNameAndOrg}</div>
```

Comment:
`span` er mer egnet her siden `button` er inline, og man bør derfor ikke ha blokkelementer som `div` inni den.
```suggestion
      <span className={classes.userOrgNames}>{userNameAndOrg}</span>
```